### ISSUE TITLE
Feature request: visual indicator for ongoing tx/rx on a port 

### ISSUE THREAD

**Issue by garykuo1004** [pstavirs/ostinato#269]
It would be great to have visual indicator for ongoing tx/rx on a port (indicator like Smartbits led).

**Comment by hansfn** [comment]
Yes, and logging to the new logging (Logs) window when a stream starts and ends.

**Comment by pstavirs** [comment]
There is already a port transmit/capture state in the port stats window -
![capture](https://user-images.githubusercontent.com/8395079/44667902-b0bf8780-aa39-11e8-8188-132c8527900a.PNG)

Would converting the `On/Off` text to graphical icons be better? Any specific reason why it should go to logs window?

**Comment by hansfn** [comment]
Yes, I guess green/red lights or something, is easier to spot, however, thinking about it I'm fine with the textual on/off state ;-) 

Regarding logging (which is a separate issue really): When I have multiple streams - that are run in sequence, it would be nice to know when one stream (starts and) ends and another starts. When I first tried Ostinato it would have made me more sure that I was doing it right ;-)

**Comment by pstavirs** [comment]
@hansfn Green/Red light may for transmit/capture state might be a bit confusing 'coz those are used to inidicate port oper state. Will think about other icons.

Logging start of each stream during transmit could become complicated given the current implementation. Unless folks come up with one or more convincing use cases for the same, I won't consider implementation just yet.
